### PR TITLE
Feature/37614 New school home

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
- [x] New feature toggle for updated school home
- [x] Govuk navigation component
- [x] New school home view
- [x] View Functionality

Partial screen:

Screenshot 2019-10-30 at 14 27 03

### CODE REVIEW

File: admin/package.json [hunk 1]
```diff
@@ -78,6 +78,9 @@
     "xregexp": "^4.2.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.6.4",
+    "@babel/node": "^7.6.3",
```
Comment (GuyHarwood) [inline admin/package.json]: as discussed, please remove

File: admin/views/school/school-home-v2.ejs [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+                    </p>
+                    <% } %>
+                </li>
+            </ol>
+            <h3 class="govuk-heading-m govuk-!-margin-top-9">Review the check</h3>
+            <ol class="govuk-list">
+                <li>
+                    <h2 class="govuk-heading-s arrow-up govuk-!-margin-bottom-2"><a class="govuk-link" href="">Pupil status</a></h2>
+                    <p class="govuk-body-m">
+                        View the pupil status after they have taken te check
```
Comment (jon-shipley) [inline admin/views/school/school-home-v2.ejs]: typo 'te'

File: admin/package.json [hunk 3]
```diff
@@ -78,6 +78,9 @@
...
+    "@babel/preset-env": "^7.6.3",
```
Comment (jon-shipley) [inline admin/package.json]: I may be missing something here.  Where are you specifying the target output for babel?

File: admin/views/school/school-home-v2.ejs (hunk 2 above)
Comment (GeorgeChatzigiannis) [reply admin/views/school/school-home-v2.ejs]: @jon-shipley resolved

File: admin/package.json (hunk 1 above)
Comment (GeorgeChatzigiannis) [reply admin/package.json]: done 👍

**Pull request merged**